kinesis-cloud/read-data.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Source library
+. ../utils/helper.sh
+
+source config/demo.cfg
+DELTA_CONFIGS_DIR=delta_configs
+source $DELTA_CONFIGS_DIR/env.delta
+
+check_env || exit 1
+check_aws || exit
+
+echo -e "\n\nKinesis data:"
+while read -r line ; do echo "$line" | base64 -id; echo; done <<< "$(aws kinesis get-records --shard-iterator $(aws kinesis get-shard-iterator --shard-id shardId-000000000000 --shard-iterator-type TRIM_HORIZON --stream-name $KINESIS_STREAM_NAME --query 'ShardIterator') | jq '.Records[].Data')"
```

Comment:
Might be good for this command to specify the region configured in `config/demo.cfg` in case the user's default region does not match

Reply:
Agree, and done!